Server/ChatService/main.py

```
@@ -97,6 +98,24 @@ def create_app() -> FastAPI:
+    # Add validation error handler for better debugging
+    from fastapi.exceptions import RequestValidationError
+    from fastapi.responses import JSONResponse
+    
+    @app.exception_handler(RequestValidationError)
+    async def validation_exception_handler(request: Request, exc: RequestValidationError):
+        """Handle validation errors and log them"""
+        print(f"[validation] Error on {request.url.path}: {exc.errors()}")
+        try:
+            body = await request.body()
+            print(f"[validation] Request body: {body.decode('utf-8') if body else 'empty'}")
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)